src/plugins/pluginHooks.ts

```diff
@@ -10,6 +10,11 @@ export const usePluginState = <T>(name: string) => {
   return pair;
 };
 
+export const useSinglePluginSlot = (name: string) => {
```

Comment:
* Maybe `usePluginSlot` or `usePluginSingleSlot`, to keep it prefixed with `usePlugin` namespace.
   * `usePluginSlot` makes rather more sense, as `usePluginSlotList` is specific, claiming that "single" is default
* It should either ensure that there is only single element, or use priority - `const usePluginSlot = (name: string) => usePluginSlotList(name)[0];`